agent/src/beerocks/slave/beerocks_slave_main.cpp

```diff
@@ -168,7 +169,7 @@ static void fill_son_slave_config(beerocks::config_file::sConfigSlave &beerocks_
     son_slave_conf.backhaul_wireless_iface_filter_low =
         beerocks::string_utils::stoi(beerocks_slave_conf.sta_iface_filter_low[slave_num]);
     son_slave_conf.backhaul_wireless_iface_type = son_slave_conf.hostap_iface_type;
-    son_slave_conf.no_vendor_specific           = beerocks_slave_conf.no_vendor_specific == "1";
+    // son_slave_conf.no_vendor_specific = 
```

Comment:
Never leave something commented-out. Just remove the code.

Reply:
Also, for completeness, the no_vendor_specific option should be removed from sSlaveConfig and moved to another structure.